instance/models/instance.py

```diff
@@ -495,7 +527,15 @@ class OpenEdXInstance(AnsibleInstanceMixin, GitHubInstanceMixin, Instance):
     s3_secret_access_key = models.CharField(max_length=50, blank=True)
     s3_bucket_name = models.CharField(max_length=50, blank=True)
 
-    ANSIBLE_SETTINGS = AnsibleInstanceMixin.ANSIBLE_SETTINGS + ['ansible_s3_settings']
+    ephemeral_databases = models.BooleanField()
```

Comment:
@omarkhan `use_ephemeral_databases`, to show that it's a boolean rather than DB objects?